frontend/README.md

```diff
@@ -103,12 +103,8 @@ This should keep you out of CORS troubles
 ![localhost serve](docs/localhost_debugging.png)
 
 # Docker local development
+By default the front-end will be running in `development mode`, consequently make sure you setup the environment files as described [here](###-Modify-Environment-Variables).
 ```
 docker build -t linkylink-fe .
-docker run -it --rm -p 8080:8080 \
--e "VUE_APP_BACKEND=http://localhost:5000" \
--e "VUE_APP_OIDC_AUTHORITY=https://<b2c login subdomain>.onmicrosoft.com/v2.0/.well-known/openid-configuration?p=B2C_1_SignUp_SignIn" \
--e "VUE_APP_OIDC_CLIENT_ID=<client id?" \
--e "VUE_APP_OIDC_SCOPE=openid https://testprodoh.onmicrosoft.com/api/UrlBundle.ReadWrite" \
-linkylink-fe
+docker run -it --rm -p 8080:8080 --name frontend linkylink-fe
```

Comment:
Is the idea that you would have all your env vars set in the `.env` file before running the container vs passing them in on the CLI ?

Reply:
I got it working with passing the env variables as params, so I am going to reverse this.